spec/steps/articles/articles_list_steps.rb

```diff
@@ -3,6 +3,16 @@ module ArticlesListSteps
 
   # WHEN
 
+  step 'I click on article in recent post on article list page' do
+    s = self
```

Comment:
why do we need it? we could use just `article.title` or `self.article.title`

Reply:
ok, it is more clear for me now. We deal with 2 different contexts. But then I recommend to use following:

```ruby
article_title = article.title
ArticleListPage.on { open_recent_post(article_title) }
```

Reply:
on 3rd revision I found we have it everywhere, but it looks ugly. Let's create separate task for that.